src/applications/financial-status-report/tests/unit/transform.unit.spec.jsx

```diff
@@ -0,0 +1,1106 @@
+        expect(submissionObj.assets.automobiles[0]).haveOwnProperty(
+          'resaleValue',
+        );
+      });
+      it('has valid data', () => {
+        const submissionObj = JSON.parse(transform(null, inputObject));
+        expect(submissionObj.assets.automobiles[0].make).to.equal('Dodge');
+        expect(submissionObj.assets.automobiles[0].model).to.equal('RAM');
+        expect(submissionObj.assets.automobiles[0].year).to.equal('2020');
+        expect(submissionObj.assets.automobiles[0].resaleValue).to.equal(
+          '75000.45',
+        );
+      });
+    });
+    describe('otherAssets', () => {
+      it('has valid structure', () => {
+        const submissionObj = JSON.parse(transform(null, inputObject));
+        expect(submissionObj.assets.otherAssets[0]).haveOwnProperty('name');
+        expect(submissionObj.assets.otherAssets[0]).haveOwnProperty('amount');
+      });
+      it('has valid data', () => {
+        const submissionObj = JSON.parse(transform(null, inputObject));
+        expect(submissionObj.assets.otherAssets[0].name).to.equal('Bitcoin');
+        expect(submissionObj.assets.otherAssets[0].amount).to.equal('50000.57');
+      });
+    });
+  });
+  describe('installmentContractsAndOtherDebts', () => {
+    it('has valid structure', () => {
+      const submissionObj = JSON.parse(transform(null, inputObject));
+      expect(submissionObj).haveOwnProperty(
+        'installmentContractsAndOtherDebts',
+      );
+      expect(submissionObj.installmentContractsAndOtherDebts).is.an('array');
+    });
+    describe('installmentContractsAndOtherDebts list item', () => {
+      it('has valid structure', () => {
+        const submissionObj = JSON.parse(transform(null, inputObject));
+        expect(submissionObj.installmentContractsAndOtherDebts[0]).is.an(
+          'object',
+        );
+        expect(
+          submissionObj.installmentContractsAndOtherDebts[0],
+        ).haveOwnProperty('purpose');
+        expect(
+          submissionObj.installmentContractsAndOtherDebts[0],
+        ).haveOwnProperty('creditorName');
+        expect(
+          submissionObj.installmentContractsAndOtherDebts[0],
+        ).haveOwnProperty('originalAmount');
+        expect(
+          submissionObj.installmentContractsAndOtherDebts[0],
+        ).haveOwnProperty('unpaidBalance');
+        expect(
+          submissionObj.installmentContractsAndOtherDebts[0],
+        ).haveOwnProperty('amountDueMonthly');
+        expect(
+          submissionObj.installmentContractsAndOtherDebts[0],
+        ).haveOwnProperty('dateStarted');
+        expect(
+          submissionObj.installmentContractsAndOtherDebts[0],
+        ).haveOwnProperty('amountPastDue');
+        expect(
+          submissionObj.installmentContractsAndOtherDebts[0],
+        ).haveOwnProperty('creditorAddress');
+        expect(
+          submissionObj.installmentContractsAndOtherDebts[0].creditorAddress,
```

Comment:
Does this object have any properties that we care about/ are tested in other places?